sound/soc/sof/sof-priv.h

```diff
@@ -383,6 +387,17 @@ enum snd_sof_fw_state {
 	SOF_FW_BOOT_COMPLETE,
 };
 
+struct sof_ipc4_fw_modules {
+	char name[SOF_MODULE_NAME_LEN];
+	u32	id;
+	u8 uuid[UUID_SIZE];
+	u32 type;
+	u32	*instance_id; /* bit array */
+	u16 instance_max_count;
+	u32 bss_size;
+	void *private;
```

Comment:
align to a tab or keep a single space, don't add random spaces.

Reply:
ack